elasticsearch/includes/classes/Health.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Automattic\VIP\Elasticsearch;
+
+use \ElasticPress\Indexable as Indexable;
+use \ElasticPress\Indexables as Indexables;
+
+use \WP_Query as WP_Query;
+use \WP_User_Query as WP_User_Query;
+
+
+class Health {
+	/**
+	 * Verify the difference in number for a given entity between the DB and ElasticSearch.
+	 * Entities can be either posts or users.
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 * @param array $query_args Valid WP_Query criteria, mandatory fields as in following example:
+	 * $query_args = [
+	 *		'post_type' => $post_type,
+	 *		'post_status' => array( $post_statuses )
+	 * ];
+	 *
+	 * @param mixed $indexable Instance of an ElasticPress Indexable Object to search on
```

Comment:
Instead of `mixed`, we can reference the base class directly
```suggestion
	 * @param \ElasticPress\Indexable $indexable Instance of an ElasticPress Indexable Object to search on
```